modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -179,6 +179,14 @@ enum class Feature(
         hasFirebaseRemoteFlag = true,
         hasDevToggle = true,
     ),
+    PODCAST_HTML_DESCRIPTION(
+        key = "podcast_html_description",
+        title = "Use HTML in the podcast description",
+        defaultValue = true,
+        tier = FeatureTier.Free,
+        hasFirebaseRemoteFlag = true,
```

Comment:
Since you are already setting this with the firebase flag, I am adding this comment just as a reminder to add the toggle in firebase